drivers/ethernet/Kconfig.native_posix

```diff
@@ -12,6 +12,27 @@ menuconfig ETH_NATIVE_POSIX
 	  a process in your host system.
 
 if ETH_NATIVE_POSIX
+config ETH_NATIVE_POSIX_STARTUP_AUTOMATIC
+	bool "Start network interface automatically"
+	help
+	  If set, then the native_posix ethernet driver tries to setup network
+	  interface by itself. This can be annoying as then you would need to
+	  execute the zephyr.exe as a root user. Recommended way is to start
```

Comment:
One benefit of starting the interface manually is that the interface is permanent, meaning it remains after echo server is stopped. The advantage of this is that wireshark will be left running between zephyr restarts and initial packets will be captured too. Hence, using CONFIG_ETH_NATIVE_POSIX_STARTUP_SCRIPT to start wireshark won't be needed. IMHO this benefit could be mentioned too.

Reply:
Well, given that the manual start is new default, and this help text is already long, I guess we shouldn't add wireshark howto here.

(I for example would think that it this description "thumbs down" running as root, it should mention that net-setup.sh will certainly require root too - but well, we don't write a magazine article here.)